app.rb

```diff
@@ -1,9 +1,12 @@
 require 'sequel'
 require 'sinatra/base'
+require './lib/loader'
 require './lib/mac_formatter.rb'
```

Comment:
Be nice to move these requires into the `loader.rb`

Reply:
WIll add `require_all` next.